org.eclipse.scanning.test/src/org/eclipse/scanning/test/scan/AbstractScanTest.java

```diff
@@ -216,7 +216,7 @@ public void testStepScan() throws Exception {
 		model.setStep(1);
 		model.setName("myScannable");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "myScannable" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)